src/libraries/System.Data.Common/src/System/Data/Filter/Operators.cs

```diff
@@ -229,7 +229,7 @@ internal static string ToString(int op)
         {
             string st;
 
-            if (op <= s_looks.Length)
```

Comment:
From a quick inspection:
a) The same issue appears elsewhere in this file, e.g. in Priority
b) This will never actually happen because the only values that can be passed in here are within range.

This code has been around for a very long time.  If we're going to modify it, I'd prefer we validate the full fix, which is likely to delete dead code in here and add a Debug.Assert.